packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -60,66 +54,15 @@ const rainbowkitTheme = merge(darkTheme(), {
     accentColor: 'var(--blue-link)'
   },
   fonts: {
-    body: "'Space Grotesk', sans-serif"
+    body: 'Roboto, sans-serif'
   }
 } as Theme)
 
-const AppContent = (): JSX.Element => {
-  const [{ sourceChain }] = useNetworks()
+const ArbTokenBridgeStoreSyncWrapper = (): JSX.Element | null => {
```

Comment:
moved the other UI logic to `AppContent` so that this component only computes `tokenBridgeParams` and return the `ArbTokenBridgeStoreSync` when ready

Reply:
this does a few more things than that: checks if cctp is blocked, handles the edge case when a selected token is native to a chain, fetches warning tokens

Reply:
That is true, not entirely sure how to encapsulate those in the component name, but I believe it'll require dedicated refactoring effort to clean this up. For now it is mostly updated to a state where the UI can be cleanly updated